### ISSUE TITLE
Fix new mkdocs issues

### ISSUE THREAD

**Issue by viceice** [renovatebot/renovatebot.github.io#457]
### What browser are you using?

Chrome

### Describe the bug

new build issues (currently only info level)

```
INFO    -  DeprecationWarning: To replace an existing file, call `remove` before `append`.
  File "/home/runner/.cache/pypoetry/virtualenvs/docs-renovatebot-com-NA9kyG5D-py3.12/lib/python3.12/site-packages/material/plugins/privacy/plugin.py", line 367, in _queue
    self.assets.append(file)
  File "/home/runner/.cache/pypoetry/virtualenvs/docs-renovatebot-com-NA9kyG5D-py3.12/lib/python3.12/site-packages/mkdocs/structure/files.py", line 100, in append
    warnings.warn(
INFO    -  Doc file 'configuration-options.md' contains a link './self-hosted-configuration.md#allowedEnv', but the doc 'self-hosted-configuration.md' does not contain an anchor '#allowedEnv'.
INFO    -  Doc file 'configuration-options.md' contains a link './self-hosted-configuration.md#allowedHeaders', but the doc 'self-hosted-configuration.md' does not contain an anchor '#allowedHeaders'.
INFO    -  Doc file 'configuration-options.md' contains a link './getting-started/running.md#githubcom-token-for-release-notes', but the doc 'getting-started/running.md' does not contain an anchor '#githubcom-token-for-release-notes'.
INFO    -  Doc file 'self-hosted-configuration.md' contains a link './self-hosted-experimental.md#renovatexs3endpoint', but the doc 'self-hosted-experimental.md' does not contain an anchor '#renovatexs3endpoint'.
INFO    -  Doc file 'self-hosted-configuration.md' contains a link './self-hosted-experimental.md#renovatexs3pathstyle', but the doc 'self-hosted-experimental.md' does not contain an anchor '#renovatexs3pathstyle'.
INFO    -  Doc file 'getting-started/private-packages.md' contains a link '../self-hosted-configuration.md#detectHostRulesFromEnv', but the doc 'self-hosted-configuration.md' does not contain an anchor '#detectHostRulesFromEnv'.
``` 

### Steps to reproduce

1. build from main

**Comment by viceice** [comment]
@squidfunk the first info message seems odd

**Comment by squidfunk** [comment]
Have you upgraded to the latest version? The issue should be fixed in 9.5.20. If not, please create a new issue with a minimal reproduction on our issue tracker, or ask on our discussion board – the best place to ask me questions 😅

**Comment by viceice** [comment]
OK,  thanks. i saw a pending upgrade. will check again after that is merged.
